licenses/licenses_tests.py

```diff
@@ -11,6 +11,9 @@
 LIST_URL = f'{DOMAIN}{reverse_lazy("licenses:licenses")}'
 CREATE_URL = f'{DOMAIN}{reverse_lazy("licenses:create")}'
 LOGIN_URL = f'{DOMAIN}{reverse_lazy("login")}'
+# the LR created first has the id 1
+DETAILS_URL = f'{DOMAIN}{reverse_lazy("licenses:details", args=[1])}'
+EDIT_URL = f'{DOMAIN}{reverse_lazy("licenses:update", args=[1])}'
```

Comment:
It would be better for the id/arg to be passed in the test via `license_request.id`. This would be a better test isolation as it is independent of the actual id of the `license_request`.